api/src/AppBundle/Service/Audit/AwsAuditLogHandler.php

```diff
@@ -0,0 +1,174 @@
+<?php declare(strict_types=1);
+
+namespace AppBundle\Service\Audit;
+
+use Aws\CloudWatchLogs\CloudWatchLogsClient;
+use Aws\CloudWatchLogs\Exception\CloudWatchLogsException;
+use Monolog\Logger;
+
+class AwsAuditLogHandler extends AbstractAuditLogHandler
+{
+    /** @var CloudWatchLogsClient */
+    private $client;
+
+    /** @var string */
+    private $group;
+
+    /** @var string */
+    private $stream;
+
+    /** @var string */
+    private $sequenceToken;
+
+    /** @var array */
+    private $existingStreams = [];
+
+    /** @var bool */
+    private $initialized = false;
+
+    /**
+     * @param CloudWatchLogsClient $client
+     * @param $group
+     * @param int $level
+     * @param bool $bubble
+     */
+    public function __construct(CloudWatchLogsClient $client, $group, $level = Logger::NOTICE, $bubble = true)
+    {
+        $this->client = $client;
+        $this->group = $group;
+
+        parent::__construct($level, $bubble);
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
Like this :D didn't even know it was a thing 👍🏻

Reply:
Yeah - one thing in PHP that encourages us to type less 🙃